### ISSUE TITLE
Multiple Attachments IE

### ISSUE THREAD

**Issue by koolaide** [osTicket/osTicket-1.7#244]
Problem adding more than 1attachments when opening a ticket using Internet Explorer, using another browser is is not an issue.

IE8 - Allows 2 attachments at a time
IE9 - Allows 1 attachment at a time

**Comment by koolaide** [comment]
Has anyone been able to verify this issue with Internet Explorer Browsers?

**Comment by TribeWeb** [comment]
Yep - got same issue.  Chrome on OSX no problem: IE9 only allows one file upload (not tested any others)  (I'm still on RC3 - will test tomorrow with RC4)
